docs/ObjectStoreAccessKeyPost.md

```diff
@@ -0,0 +1,11 @@
+# ObjectStoreAccessKeyPost
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**user** | [**Reference**](Reference.md) | Reference of the associated user. | [optional] 
+**secret_access_key** | **str** | The secret access key to import from another FlashBlade. To import a set of credentials, this field must be specified with the &#x60;names&#x60; query parameter. If both of these not specified, the system will generate a new set of credentials. | [optional] 
```

Comment:
I think this is generated by another file, so probably my fault, but small typo here `If both these not specified` -> `If both of these are not specified`